docs/guide/algob-console.md

```diff
@@ -10,127 +10,357 @@ Sometimes it's nice to work with your contracts interactively for testing and de
+      212, 234,  17,  11, 212, 103, 128, 253,  57,  65, 153,
+       47,  59, 220, 182, 226, 212,  35,  72, 248, 103, 123,
+       76,   1, 223, 193, 196, 117, 231, 147,  45,  80,  53,
+      104, 133, 229, 135, 144, 198,  21, 238,  73, 253, 177,
+      135, 114, 142,  42,  36,  42,  97, 130,  11
+    ],
+    txn: {
+      amt: 1000000,
+      fee: 257000,
+      fv: 3725,
+      gen: 'private-v1',
+      gh: [Uint8Array],
+      lv: 4725,
+      note: [Uint8Array],
+      rcv: [Uint8Array],
+      snd: [Uint8Array],
+      type: 'pay'
+    }
+  }
+}
 ```
-![image](https://user-images.githubusercontent.com/33264364/97816714-08627900-1cbd-11eb-86db-1dffbceb125f.png)
 
-## Transfer Assets
+![image](https://user-images.githubusercontent.com/33264364/122478061-5a56d580-cfe6-11eb-9bb4-7e2329dd7037.png)
 
-We will transfer a single unit of Algorand Standard Asset `gold`(which we deployed during setup) from `goldOwnerAccount` to `johnAccount` using `transferAsset` function present within `algob` context (`algob.transferAsset(..)`). Code can be found in `/scrips/transfer/gold-to-john.js`.
+## Transfer Assets
 
+We will transfer a single unit of Algorand Standard Asset `gold`(which we deployed during setup) from `goldOwner` to `john`. Relevant code can be found in `/scrips/transfer/gold-to-john.js`.
+
+Let's use `.editor` mode of REPL to write & execute multiple lines of code at once:-
+```bash
+algob> .editor
+// Entering editor mode (Ctrl+D to finish, Ctrl+C to cancel)
+const rtypes = algob.runtime.types;
+const gold = deployer.asa.get('gold'); // asa info from checkpoint
+const goldOwner = deployer.accountsByName.get('alice');
+const john = deployer.accountsByName.get('john');
+algob.executeTransaction(deployer, {
+  type: rtypes.TransactionType.TransferAsset,
+  sign: rtypes.SignType.SecretKey,
+  fromAccount: goldOwner,
+  toAccountAddr: john.addr,
+  amount: 1,
+  assetID: gold.assetIndex,
+  payFlags: {}
+});
 ```
-goldAssetID = deployer.asa.get("gold").assetIndex
-await algob.transferAsset(deployer, goldAssetID, goldOwnerAccount, johnAccount.addr, 1)
-```
-![image](https://user-images.githubusercontent.com/33264364/97816941-3e086180-1cbf-11eb-87f1-485bde49323c.png)
+![image](https://user-images.githubusercontent.com/33264364/122479488-c8040100-cfe8-11eb-8f42-291459fd6c69.png)
 
-You can also check balance of `johnAccount` using `algob.balanceOf(..)`
-```
-await algob.balanceOf(deployer, johnAccount.addr, goldAssetID);
+After transferring ASA, you can also check the balance (asset holding) of `john` using `algob.balanceOf(..)`
+```bash
+algob> await algob.balanceOf(deployer, john.addr, gold.assetIndex);
 ```
+![image](https://user-images.githubusercontent.com/33264364/122479899-7a3bc880-cfe9-11eb-99f2-e7ad70e2646f.png)
 
-Similar example - `/scrips/transfer/tesla-to-john.js`
+Similar example can be found in `/scrips/transfer/tesla-to-john.js` (tesla ASA).
 
 ## Transfer Algos according to ASC logic (Contract Account)
```

Comment:
i couldn't find the link to file for this section, Please add it if not  added.
Ex: ``/scrips/transfer/tesla-to-john.js``

Reply:
yes, it's in `/scripts/transfer/gold-contract-sc.js`. We have a reference at the bottom of this section - [https://github.com/scale-it/algo-builder/pull/385/files/87c26bd2d7de5baee525f268e609e3d0eb5948b1#diff-fb6c1b787cf243abef74cf124766885f78c38a3081ccb2b5ff3ad0b52498b7e5R280](https://github.com/scale-it/algo-builder/pull/385/files/87c26bd2d7de5baee525f268e609e3d0eb5948b1#diff-fb6c1b787cf243abef74cf124766885f78c38a3081ccb2b5ff3ad0b52498b7e5R280)